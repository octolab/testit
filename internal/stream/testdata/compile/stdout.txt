FAIL	testdata/nocompile [build failed]
?   	testdata/nofiles	[no test files]
ok  	testdata/notests	0.620s [no tests to run]
ok  	testdata/panicked	0.425s [no tests to run]
FAIL
