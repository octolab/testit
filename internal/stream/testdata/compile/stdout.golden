FAIL	testdata/nocompile [build failed]
FAIL
